test/functional/mempool_package_limits.py

```diff
@@ -339,12 +317,11 @@ def test_desc_size_limits(self):
         and in-package descendants are all considered together.
         """
         node = self.nodes[0]
-        assert_equal(0, node.getmempoolinfo()["size"])
         target_weight = 21 * 1000 * WITNESS_SCALE_FACTOR
-        high_fee = Decimal("0.0021") # 10 sats/vB
+        high_fee = 210_000  # 10 sats/vB
```

Comment:
To make this work, we need to convert back from weight to vsize first. Since `fee = (target_weight / WITNESS_SCALE_FACTOR) * 10` looks rather weird, I decided to specify the target size directly in vsize and deduce both fee and weight from that. One more variable, but (hopefully) more readable and less indirect magic number magic involved.